python/ray/serve/application.py

```diff
@@ -0,0 +1,190 @@
+from typing import List, Dict, TextIO, Optional, Union
+import json
+import yaml
+import time
+
+import ray
+from ray import serve
+from ray.serve.api import (
+    Deployment,
+    _get_global_client,
```

Comment:
if serve global client is accessed and imported outside of its own file, should we make it non-private instead ?

Reply:
That's a good question– I'm ok with it, but I'm also wondering if this reveals an anti-pattern in the implementation. cc @edoakes

Reply:
maybe call it `internal_get_global_client`? the original reason was not to expose this to user.